src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/ReadDataProgramGroup.java

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.hellbender.cmdline.programgroups;
+
+import org.broadinstitute.barclay.argparser.CommandLineProgramGroup;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+
+public class ReadDataProgramGroup implements CommandLineProgramGroup {
```

Comment:
I don't understand what this group is, the description makes it sound like it should be `IntervalProgramGroup`

"Tools that process genomic intervals in various formats"

Reply:
Thats totally a mistake...

Reply:
Can we add an `s` to make _IntervalsProgramGroup_ please.